### DISCUSSION TITLE
[Feature] Be able to add folders in external libraries and have them appear as albums

### DISCUSSION THREAD

**Discussion by EdwardK9** [immich-app/immich#14849] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be awesome to be able to create an external libraries and then copy folders and files into there. Immich could them import the files as files and import the folders as albums.
I know I would love this.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4279
